### DISCUSSION TITLE
Issue with bootstrap/cache folder

### DISCUSSION THREAD

**Discussion by CharlyMollard** [livewire/livewire#5583] [7. Bugs (archived)] [answered]
Hi, my laravel project give me an error that said `The C:\Travail\Web\_Servers\BARBOFUS\bootstrap\cache directory must be present and writable.`, to clear this error I have to manually remove the cache folder and recreate it empty after each branch creation. The error occure only on the pages that contains livewire code. I tried creating a new laravel 10 project, and as soon as I installed livewire, the same issue appeared. If I do the commande `php artisan cache:clear` I got a `Failed to clear cache. Make sure you have the appropriate permissions.`.  I used livewire before (like one month ago) on the same project, and I never had this problem, it appeared after I update livewire version, but I don't know how to know what version I had before.

- composer v2.5.4
- Laravel v9.52.4
- Livewire v2.12.1
- PHP v8.2.3.

**Comment by lrljoe** [comment, marked as answer]
Check the filesystem owner on the bootstrap directory. 

Ensure permissions are set correctly to allow for the directory to be written by the user that the webserver is running as.

**Reply by CharlyMollard** [reply]
Hi, how can I do this ? I see a lot of tutorial on the web but they are on a webserver, I'm currently developing the website, it is not host, i'm on local with Laragon. (and windows 10)

**Reply by lrljoe** [reply]
Use the terminal option in laragon and it's the same instructions as online for other server instances.

Or if you just set the folders to be writable by "everyone" if you're only doing some local dev/tinkering, just don't do that if you are sharing it externally.

**Reply by CharlyMollard** [reply]
Ok, for the full solution :
I follow [those command](https://www.itsolutionstuff.com/post/how-to-setup-file-permissions-correctly-in-laravelexample.html) to change the permissions, and installed sudo on windows with the help in [this link](https://linuxhint.com/run-sudo-command-windows/).
Thanks for the answers !
